[Minor] Update Super-Kamiokande description (#3248)
nasa-gcn/gcn.nasa.gov: app/routes/missions.sksn/route.mdx

**Start:** 1996

**Data Archives:**
[University of Tokyo](https://www-sk.icrr.u-tokyo.ac.jp/en/sk/for-reseacher/)

> The [Super-Kamiokande experiment](https://www-sk.icrr.u-tokyo.ac.jp/en/sk/), also known as Super-K or SK, was built and has been operated with funding from the Japanese Ministry of Education, Science, Sports and Culture, and the U.S. Department of Energy.

Reviewer: ```suggestion
The [Super-Kamiokande experiment](https://www-sk.icrr.u-tokyo.ac.jp/en/sk/), also known as Super-K or SK, was built and is operated with funding from the Japanese Ministry of Education, Science, Sports and Culture, and the U.S. Department of Energy.
```

> The Hyper-Kamiokande experiment will be the successor of SK, it is expected to start operation around 2028.

Reviewer: ```suggestion
The Hyper-Kamiokande experiment will be the successor of SK. It is expected to start operation around 2028.
```
